### PR TITLE
fix(odyssey-react-mui): Add min-heights to keep correct heights

### PR DESCRIPTION
**Pull request by bryancunningham-okta**
[OKTA-795987](https://oktainc.atlassian.net/browse/OKTA-795987)

## Testing & Screenshots

- [ ] I have confirmed this change with my designer and the Odyssey Design Team.

### CODE REVIEW

**Review by jordankoschei-okta — DISMISSED** [review]
Clever fix, lgtm 👍

File: packages/odyssey-react-mui/src/Select.tsx [hunk 1]
```diff
@@ -417,6 +417,7 @@ const Select = <
 
         return (
           <MuiMenuItem
+            data-empty={Boolean(!option.text)}
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-react-mui/src/Select.tsx:420]: 😆

```suggestion
            data-empty={!option.text}
```

File: packages/odyssey-react-mui/src/theme/components.tsx [hunk 2]
```diff
@@ -2172,6 +2172,13 @@ export const components = ({
             },
           },
 
+          "&[data-empty='true']": {
+            "&:after": {
+              content: "'&nbsp;'",
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-react-mui/src/theme/components.tsx:2177]: Do we specifically want a non-breaking space here?

**Review by KevinGhadyani-Okta — DISMISSED** [review]
One comment. The rest looks fine.

**Review by jordankoschei-okta — APPROVED** [review]

**Pull request merged**